health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,333 @@
+  const init = (id, defaultBaseMap) => {
+    if (mapRef.current) return;
+
+    let mapConfig = {
+      center: [lat || 0, long || 0],
+      zoomControl: false,
+      zoom: 12,
+      scrollwheel: true,
+      minZoom: 3,
+    };
+
+    let map_i = L.map(id, mapConfig);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'map_i' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)